src/app/shared/menu/date/custom-date-range-dropdown.component.css

```diff
@@ -44,6 +44,15 @@
   min-width: 285px;
 }
 
+@media (max-width: 768px) {
```

Comment:
This menu doesn't fit in small screen sizes, so I decided to not display the redundant controls for the list of standard ranges until we can prioritize responsive and mobile design. :deal-with-it-parrot:

Reply:
confirmed, looks ok

![screen shot 2018-02-16 at 11 14 25 am](https://user-images.githubusercontent.com/1205061/36320218-1066d06a-130b-11e8-9ca6-bbf9753dfffb.png)

Reply:
👍 Yep, much better as a temporary measure at least. Google Analytics reports almost 20% mobile users.